electron/main.ts

```diff
@@ -29,11 +25,11 @@ import i18next from 'i18next'
 import isDev from 'electron-is-dev'
 
 import { DXVK } from './dxvk'
+import { Game } from './games'
 import { GameConfig } from './game_config'
 import { GlobalConfig } from './config'
-import { LegendaryGame } from './games'
-import { Library } from './legendary_utils/library'
-import { User } from './legendary_utils/user'
+import { LegendaryLibrary } from './legendary/library'
```

Comment:
I'm wondering: should we really use Legendary or should we use Epic? EpicUser, EpicGame. I think makes more sense to me since Legendary is the backend now, but we might change and implement our own in the future.

Reply:
Yeah, that would have its own seperate folder.